### PR TITLE
[14.0] [MIG] sale_oreder_line_price_history : Migrated the module from 13 to 14.

### PR DESCRIPTION
**Pull request by dhiren-scs**

### CODE REVIEW

**Review by danimv5 — APPROVED** [review]
Functional review. LGTM!!

File: sale_order_line_price_history/static/description/index.html [hunk 1]
```diff
@@ -411,10 +411,10 @@ <h1><a class="toc-backref" href="#id2">Known issues / Roadmap</a></h1>
 </div>
 <div class="section" id="bug-tracker">
 <h1><a class="toc-backref" href="#id3">Bug Tracker</a></h1>
-<p>Bugs are tracked on <a class="reference external" href="https://github.com/OCA/sale-workflow/issues">GitHub Issues</a>.
+<p>Bugs are tracked on <a class="reference external" href="https://github.com/OCA/timesheet/issues">GitHub Issues</a>.
```
Comment (dsolanki-initos) [inline sale_order_line_price_history/static/description/index.html:414]: why it is replaced with timesheet?

File: sale_order_line_price_history/static/description/index.html [hunk 2]
```diff
@@ -411,10 +411,10 @@ <h1><a class="toc-backref" href="#id2">Known issues / Roadmap</a></h1>
...
 In case of trouble, please check there if your issue has already been reported.
 If you spotted it first, help us smashing it by providing a detailed and welcomed
-<a class="reference external" href="https://github.com/OCA/sale-workflow/issues/new?body=module:%20sale_order_line_price_history%0Aversion:%2013.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**">feedback</a>.</p>
+<a class="reference external" href="https://github.com/OCA/timesheet/issues/new?body=module:%20sale_order_line_price_history%0Aversion:%2014.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**">feedback</a>.</p>
```
Comment (dsolanki-initos) [inline sale_order_line_price_history/static/description/index.html:417]: same here

**Comment by dsolanki-initos** [comment]
Functional Review done

**Comment by brendapaniagua** [comment]
Functional review done

**Comment by dsolanki-initos** [comment]
@dhiren-scs can you please check those changes?

**Comment by dsolanki-initos** [comment]
Hi @dhiren-scs ,
Is it possible to have a look at those suggested changes?

**Comment by dsolanki-initos** [comment]
Hello @dhiren-scs it would be nice if you could have a look at the above comments and do the needfull.

**Comment by dsolanki-initos** [comment]
It would be great if someone could merge the PR.

**Comment by dsolanki-initos** [comment]
It would be great if could merge the PR.

**Comment by dsolanki-initos** [comment]
It would be great if could merge the PR.

**Comment by dsolanki-initos** [comment]
It would be great if we could merge the PR.

**Comment by pedrobaeza** [comment]
We have added #1591 on v13, which is very valuable.

**Comment by dsolanki-initos** [comment]
@dhiren-scs can you please update the PR with the changes of v13.

**Comment by rousseldenis** [comment]
@dhiren-scs Do you mind if we switch to #1620 ?

**Comment by dsolanki-initos** [comment]
As we were not getting any response to the comments we have created new PR.

**Comment by rousseldenis** [comment]
> As we were not getting any response to the comments we have created new PR.

Yes, no problem. Asking to the original author is always welcome. :smiley:

**Pull request closed without merging**